config/configmap/inferenceservice.yaml

```diff
@@ -177,3 +177,8 @@ data:
         "cpuRequest": "100m",
         "cpuLimit": "1"
     }
+  defaults: |-
+    {
+        "cpu": "1",
```

Comment:
We need to differentiate request & limit?

Reply:
I was wondering if to add differentiate request & limit in the config map, but decided to follow the old hard coded logic. Agree that they should be separated, will do it.